src/app/listnews/listnews.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnInit } from '@angular/core';
+import { Listnews } from '../interfaces/listnews';
+
+@Component({
+  selector: 'app-listnews',
+  templateUrl: './listnews.component.html',
+  styleUrls: ['./listnews.component.css']
+})
+export class ListnewsComponent implements OnInit {
+
```

Comment:
adicionar propriedade para gerir a notícia highlighted com o respectivo objecto.